packages/Popover/src/Popover.js

```diff
@@ -305,29 +302,25 @@ class Popover extends React.Component {
 
   handleKeyDown = event => {
     if (event.key === "Tab" && this.isOpen() && this.$trigger) {
-      if (event.shiftKey && this.focusIsOnFirstFocusableElementInPopover()) {
+      const isFocusOnFirst = this.focusIsOnCertainElementInPopover("first") || document.activeElement === this.$content;
```

Comment:
Need to handle the case where popover is opened, then `shift+tab` is pressed immediately.